integration_tests/cleanup-jest-html-reports.sh

```diff
@@ -1,14 +1,11 @@
 #!/bin/bash
-# Set the number of directories to keep
 num_of_dir_to_keep=$1
```

Comment:
What happens if someone invokes this script without a parameter?  I wouldn't want it to remove everything.  Maybe worth checking that this has a value and exiting if it doesn't.

Reply:
You could also default to 3 if nothing is given:

`num_of_dir_to_keep=${1:-3}`